README.md

```diff
@@ -13,4 +13,4 @@ Training Management System created by [Daniel L.](https://github.com/blt950) (13
 
 Do you want to help us with improving Control Center? Curious about whether we use testing? Stylistic choices?
 
-➡️  **[Read our contribution guide](https://docs.vatsca.org/controlcenter/latest/contributing/)** for details.
+➡️  **[Read our contribution guide](https://docs.vatsca.org/controlcenter/latest/contribute/)** for details.
```

Comment:
```suggestion
➡️  **[Read our contribution guide](https://docs.vatsca.org/controlcenter/dev/contribute/)** for details.
```

The realisation that the dev (main branch) content should be used for current meta-documentation, while the features and guides are better suited to the versions contents.